src/Attribute/Duration.php

```diff
@@ -14,11 +14,11 @@ class Duration implements AttributeInterface
     private $milliseconds;
 
     /**
-     * @param $duration
+     * @param string $duration
```

Comment:
`string|int`, because the class should not know what it will be called with. In php7, you will use it as type hinting (which accepts numeric strings IIRC)